src/Components/Search/SearchBar.tsx

```diff
@@ -1,104 +1,271 @@
-import { Spacer, Spinner } from "@artsy/palette"
-import { SearchBarQuery } from "__generated__/SearchBarQuery.graphql"
+import { Box, Flex } from "@artsy/palette"
+import { SearchBar_viewer } from "__generated__/SearchBar_viewer.graphql"
+import { SearchBarSuggestQuery } from "__generated__/SearchBarSuggestQuery.graphql"
 import { ContextConsumer, ContextProps } from "Artsy"
 import Input from "Components/Input"
-import { SearchSuggestionsFragmentContainer as SearchSuggestions } from "Components/Search/Suggestions"
+import { SearchPreview } from "Components/Search/Previews"
+import { SuggestionItem } from "Components/Search/Suggestions/SuggestionItem"
 import { throttle } from "lodash"
 import React, { Component } from "react"
-import { graphql, QueryRenderer } from "react-relay"
+import Autosuggest from "react-autosuggest"
+import {
+  createRefetchContainer,
+  graphql,
+  QueryRenderer,
+  RelayRefetchProp,
+} from "react-relay"
 import styled from "styled-components"
+import { get } from "Utils/get"
+import { Media } from "Utils/Responsive"
+
+const AutosuggestContainer = styled(Box)`
```

Comment:
cc @damassi 

I was a bit unsure how to target/overwrite the CSS for this component, using our styled components setup. It seems like we need to create a wrapper/container, and then do all our targeting inside that one? After much futzing this was the only way I was able to start doing that (remove the annoying bullet points for the list styling).

Reply:
This is the right way to do it!

Reply:
If you need to overwrite a non-styled-component, like 

const Foo = props => {
  return ...
}
```
can do 
```
const StyledFoo = styled(Foo)` ... `
...
const Foo = props => {
  return <div className={props.className}>...
}
```
That will pass styles into the component